extensions/amp-lightbox-gallery/0.1/test-e2e/test-open-close.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const pageWidth = 800;
+const pageHeight = 600;
+
+describes.endtoend(
+  'AMP Lightbox Gallery Open/Close',
+  {
+    testUrl:
+      'http://localhost:8000/test/manual/amp-lightbox-gallery-launch.amp.html',
+    initialRect: {width: pageWidth, height: pageHeight},
+    // TODO(sparhami) Get this working in other environments.
+    environments: ['single'],
```

Comment:
what happens when it runs in the others? is it something to do with screen sizing or is it something else?

Reply:
I didn't dig too deeply, but when I ran locally, the page didn't show up in the viewer or shadow environments. I think it could be something about my local config though.